models/intermediate/int_amazon_selling_partner__item.sql

```diff
@@ -0,0 +1,179 @@
+with item_summary as (
```

Comment:
Shouldn't this be dependent on the `amazon_selling_partner__using_catalog_module` variable?

Reply:
yep, added